Common/UnitDefinitions/Speed.json

```diff
@@ -313,6 +313,22 @@
           "Abbreviations": [ "миль/ч" ]
         }
       ]
+    },
+    {
+      "SingularName": "Mach",
+      "PluralName": "Mach",
+      "FromUnitToBaseFunc": "{x} * 340.29",
+      "FromBaseToUnitFunc": "{x} / 340.29",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "Ma" ]
```

Comment:
It seems both "M" and "Ma" are used, I don't know which one is more widely used, but we should add both here and the most widely used one should be the first in the array, since that will be used by `ToString()`.